COPYRIGHT.md

```diff
@@ -0,0 +1,11 @@
+# Copyright
+
+Almost all code is `Copyright (c) 2019-2020 Edgeware AB`.
+
+Some code in pkg/mp4 comes from or is based on https://github.com/jfbus/mp4 which has
+`Copyright (c) 2015 Jean-François Bustarret`.
+
+Some code in pkg/bits comes from or is based on https://github.com/tcnksm/go-casper/tree/master/internal/bits
+`Copyright (c) 2017 Taichi Nakashima`.
+
+All code is [MIT license](LICENSE.txt)
```

Comment:
Why do we need both LICENSE and COPYRIGHT?

Reply:
Well, the badge script did not detect my mixed file, so I thought about separating them. And I need to keep the copyright about the code I have included.

Reply:
> And I need to keep the copyright about the code I have included.

Why? Since they're both MIT (supposedly), you don't have to.

Reply:
There is only one license, but the copyright is still not only Edgeware, so I must put it somewhere.
Anyway, the license badge still says "not identifiable by github", so next step is to add a Copyright line in the License file since that is the only difference I can see compared to Github's template.

Reply:
I pushed a new version, and now it looks fine. Added the additional copyrights in the README instead.